public/js/dashboardCategories.js

```diff
@@ -68,6 +68,56 @@ window.deleteCategory = function deleteCategory(categoryID) {
   console.log('deleted' + categoryID);
 };
 
+const inputBoxTemplate = `
+  <div id="{{instructionID}}">
+    <input id="{{inputID}}" name="{{inputID}}" type="text"></input>
+    <button id="{{removeButtonID}}" onclick="removeInstructionInput(this.id)" hidden>-</button>
+    <button id="{{addButtonID}}" onclick="addInstructionInput(this.id)">+</button><br>
+  </div>
+`;
+
+/**
+ * Adds an input box to the correct section of the page.
+ * @param {string} prevInstructionID ID of the previous input
+ * box's add button [instructionType]-add-[instructionIndex].
+ */
+function addInstructionInput(prevInstructionID) {
+  const prevInstructionIdArr = prevInstructionID.split('-');
+  const instructionType = prevInstructionIdArr[0];
+  const currInstructionIndex = parseInt(prevInstructionIdArr[2]) + 1;
+
+  const prevRemoveButton = document.getElementById(
+    `${instructionType}-remove-${prevInstructionIdArr[2]}`
+  );
```

Comment:
The parenthesis shouldn't be on a newline by itself. Is this something the formatter is doing?

Reply:
yes, not sure if there is a way to fix this.